test/test_utils/__init__.py

```diff
@@ -443,6 +443,17 @@ def is_dlc_cicd_context():
     return os.getenv("BUILD_CONTEXT") in ["PR", "CANARY", "NIGHTLY", "MAINLINE"]
 
 
+def is_not_dlc_image_type():
+    try:
+        return os.getenv("NOT_DLC").lower() == "true"
+    except:
+        return False
+
+
+def get_allowlist_path_for_enhanced_scan_from_env_variable():
+    return os.getenv("ALLOWLIST_PATH_ENHSCAN")
```

Comment:
What happens when this env variable is not defined?

Reply:
No, allowlist is not always required. In case this env variable is not defined, this will return None value. The allowlist handling lies within the try-except section, so it does not lead to any failure.